wp-resources/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -386,29 +414,24 @@ public static function searchEntries($posts, WP_Query $query): ?array {
         }
 
         $searchText = trim(get_search_query(false));
+		$taxonomies = Webonary_Parts_Of_Speech::GetPartsOfSpeechSelected();
+
         if ($searchText === '') {
-            $tax = filter_input(INPUT_GET, 'tax', FILTER_UNSAFE_RAW, array('options' => array('default' => '')));
-            if ($tax !== '') {
+            if (!empty($taxonomies[0])) {
```

Comment:
Should this be `!empty($taxonomies)`? The 0-th element may not always exist?

Reply:
@johntseng in PHP a variable is considered empty if it does not exist or if its value is falsy.